docs/usage.md

```diff
@@ -311,6 +271,14 @@ NXF_OPTS='-Xms1g -Xmx4g'
 
 ## Troubleshooting
 
+* Error: `The exit status of the task that caused the workflow execution to fail was: null.`
+
+Check the files are readable for the workflow.
+
+* Error:  `Session aborted -- Cause: Unable to execute HTTP request: ngi-igenomes.s3.amazonaws.com`
+
+The s3 connection reached the limitation of connection. Try to resume the analysis one hour later.
```

Comment:
Interesting, never heard this one before!

Reply:
Maybe it is not true. Maybe it is the limitation in our cluster side. I can always got it run if I submit the job one hour later. I will change the sentence to The internet connection reached the limitation.

Reply:
There _shouldn't_ be any limitation on the s3 side that I know of - I set this resource up and I did my best to make it as open as possible (the hosting fees are covered by AWS). I'm not too worried though, we will hopefully replace it with refgenie in the near future anyway. No need for changes in this documentation if you're happy 👍🏻